### PR TITLE
Use systemd if possible

### PR DESCRIPTION
**Pull request by benner**

### CODE REVIEW

**Comment by vdnt** [comment]
🚢

File: recipes/default.rb [hunk 1]
```diff
@@ -65,4 +69,28 @@
...
+end unless systemd_enabled
+
+systemd_service 'exabgp' do
+  description 'ExaBGP service'
+  after node[:exabgp][:systemd][:after]
+  condition_path_exists '/etc/exabgp/exabgp.conf'
+
+  service do
+    environment 'exabgp_daemon_daemonize' => 'false'
+    exec_start '/usr/src/exabgp/sbin/exabgp /etc/exabgp/exabgp.conf'
```
Comment (ernestas-vinted) [inline recipes/default.rb:81]: extract to default attributes

File: recipes/default.rb [hunk 2]
```diff
@@ -65,4 +69,28 @@
 
 runit_service 'exabgp' do
   default_logger true
+end unless systemd_enabled
+
+systemd_service 'exabgp' do
+  description 'ExaBGP service'
+  after node[:exabgp][:systemd][:after]
+  condition_path_exists '/etc/exabgp/exabgp.conf'
```
Comment (ernestas-vinted) [inline recipes/default.rb:77]: extract to default attributes

**Comment by ernestas-vinted** [comment]
exabgp-cookbook uses chefspec would be nice to few specs and output in addition

**Comment by tomas-didziokas** [comment]
🆗

File: recipes/default.rb (hunk 2 above)
Comment (ton31337) [reply recipes/default.rb:77]: `/etc/exabgp/exabgp.conf` to attributes? it's default for exabgp daemon

**Pull request merged**